senderscore.py

```diff
@@ -3,25 +3,49 @@
 
 
 class SenderScore(BotPlugin):
-    """Plugin to show the mail score of the given ip"""
+    """Plugin to show the mail score of the given address"""
 
     @botcmd
     def senderscore(self, msg, args):
-        """retrieve mail reputation usage !senderscore [IP]"""
-        return self.get_senderscores(args)
+        """Retrieve mail reputation usage !senderscore [IP/Hostname]"""
+        if self.is_ip(args):
+            ipaddresses = args
```

Comment:
Is args a list or not? Here it should be a list, but for 'hostname = args' it should be a single string right?